### PR TITLE
re-export admissions config based on recent changes

### PR DESCRIPTION
**Pull request by joewhitsitt**
# To Test

- Checkout this branch.
- `ddev composer install && ddev blt frontend && ddev drush @admissions.local sql-drop && ddev drush @admissions.local cr && ddev blt ds --site=admissions.uiowa.edu`
- There should be no config import issues.
- `ddev drush @admissions.local uli` go to https://admissions.uiowa.ddev.site/admin/structure/taxonomy/manage/tags/overview/form-display and see that both image and tvi are represented.

### CODE REVIEW

File: config/sites/admissions.uiowa.edu/config_split.patch.core.entity_form_display.taxonomy_term.tags.default.yml [hunk 1]
```diff
@@ -1,8 +1,17 @@
 adding:
+  dependencies:
+    config:
+      - field.field.taxonomy_term.tags.field_image
+    module:
+      - media_library
```
Comment (pyrello) [inline config/sites/admissions.uiowa.edu/config_split.patch.core.entity_form_display.taxonomy_term.tags.default.yml]: Okay, I know I said this was okay yesterday, but I'm realizing that these are in the base file, so they shouldn't be in the patch.

File: config/sites/admissions.uiowa.edu/config_split.patch.core.entity_form_display.taxonomy_term.tags.default.yml [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
     tvi:
       weight: 100
+    field_image:
+      type: media_library_widget
+      weight: 101
       region: content
-      settings: {  }
+      settings:
+        media_types: {  }
       third_party_settings: {  }
```
Comment (pyrello) [inline config/sites/admissions.uiowa.edu/config_split.patch.core.entity_form_display.taxonomy_term.tags.default.yml]: Ditto these.

File: config/sites/admissions.uiowa.edu/config_split.patch.core.entity_view_display.taxonomy_term.tags.default.yml [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+  content:
+    field_image:
+      type: entity_reference_entity_view
+      label: hidden
+      settings:
+        view_mode: default
+        link: false
+      third_party_settings: {  }
+      weight: 0
+      region: content
```
Comment (pyrello) [inline config/sites/admissions.uiowa.edu/config_split.patch.core.entity_view_display.taxonomy_term.tags.default.yml]: This is from the base file.

**Review by bspeare — APPROVED** [review]
All testing steps worked for me.

**Pull request merged**